test/steps/browser.js

```diff
@@ -75,6 +94,16 @@ async function thenISee(expected: string): Promise<void> {
     await assert.textIsVisible(this.driver, expected);
 }
 
+async function thenIDontSee(expected: string): Promise<void> {
+    try {
+        await assert.textIsVisible(this.driver, expected);
```

Comment:
So the test is always going to take 10s to decide the string isn't there? I guess that's acceptable but we might want to avoid making too many negative assertions like that to keep the test times down.